packages/dropdowns/src/elements/Menu/Items/AddItem.spec.tsx

```diff
@@ -31,4 +31,23 @@ describe('AddItem', () => {
 
     expect(onSelectSpy.mock.calls[0][0]).toBe('add-item');
   });
+
+  it('passes ref to underlying DOM element', () => {
+    const ref = React.createRef<HTMLLIElement>();
+
+    const { getByTestId } = render(
+      <Dropdown>
+        <Trigger>
+          <button data-test-id="trigger">Test</button>
```

Comment:
I noticed that this `data-test-id` isn't used in the test and can probably be removed.

Reply:
Removed.